.circleci/config.yml

```diff
@@ -53,41 +62,42 @@ node-core-base: &node-core-base
 node-bench-base: &node-bench-base
   resource_class: small
   working_directory: ~/dd-trace-js
+  docker:
+    - image: node
   steps:
-    - checkout
-    - *yarn-versions
-    - *restore-yarn-cache
-    - *yarn-install
-    - *save-yarn-cache
+    - checkout-and-yarn-install
     - run:
         name: Benchmark Tracer
         command: yarn bench
 
 node-bench-e2e-base: &node-bench-e2e-base
   resource_class: medium
   working_directory: ~/dd-trace-js
+
   steps:
-    - checkout
-    - *yarn-versions
-    - *restore-yarn-cache
-    - *yarn-install
-    - *save-yarn-cache
+    - checkout-and-yarn-install
     - run:
         name: E2E Benchmark
         command: yarn bench:e2e
 
 node-bench-sirun-base: &node-bench-sirun-base
+  parameters:
+    node-version:
+      type: string
+      default: "latest"
+  docker:
+    - image: node:<< parameters.node-version >>
   resource_class: small
   working_directory: ~/dd-trace-js
   steps:
-    - checkout
-    - *yarn-versions
-    - *restore-yarn-cache
-    - *yarn-install
-    - *save-yarn-cache
+    - checkout-and-yarn-install
+    - run:
+        name: Get plugin name
```

Comment:
This is already done in one of the script. It would be more consistent to do it always in the scripts or always in the CircleCI config, not both.

Reply:
Which script? AFAICT, no script is grabbing the plugin name from the `CIRCLE_JOB` env var.

Reply:
https://github.com/DataDog/dd-trace-js/pull/1505/files#diff-2e17a4cf3758f4f2c6dc40393c8d6bef4c5d77c1b67a342c2046239c8dc02ea9